app/models/samples/json/model_shoe_box.json

```diff
@@ -0,0 +1,614 @@
+                    "type": "FaceEnergyPropertiesAbridged"
+                }
+            },
+            "face_type": {
+                "type": "Wall"
+            },
+            "boundary_condition": {
+                "type": "Adiabatic",
+                "sun_exposure": "NoSun",
+                "wind_exposure": "NoWind",
+                "view_factor": "autocalculate"
+            },
+            "parent": "SimpleShoeBoxZone"
+        },
+        {
+            "type": "FaceAbridged",
+            "name": "SimpleShoeBoxZone_Top",
+            "name_original": "Simple Shoe Box Zone_Top",
+            "geometry": {
+                "type": "Face3D",
+                "boundary": [
+                    [
+                        5.0,
+                        0.0,
+                        3.0
+                    ],
+                    [
+                        5.0,
+                        10.0,
+                        3.0
+                    ],
+                    [
+                        0.0,
+                        10.0,
+                        3.0
+                    ],
+                    [
+                        0.0,
+                        0.0,
+                        3.0
+                    ]
+                ]
+            },
+            "properties": {
+                "type": "FaceProperties",
+                "energy": {
+                    "type": "FaceEnergyPropertiesAbridged"
+                }
+            },
+            "face_type": {
+                "type": "RoofCeiling"
+            },
+            "boundary_condition": {
+                "type": "Adiabatic",
+                "sun_exposure": "NoSun",
+                "wind_exposure": "NoWind",
+                "view_factor": "autocalculate"
+            },
+            "parent": "SimpleShoeBoxZone"
+        }
+    ],
+    "shades": [],
+    "apertures": [
+        {
+            "type": "ApertureAbridged",
+            "name": "SimpleShoeBoxZone_Front_Glz0",
+            "name_original": "Simple Shoe Box Zone_Front_Glz0",
+            "geometry": {
+                "type": "Face3D",
+                "boundary": [
+                    [
+                        4.5,
+                        10.0,
+                        0.7
+                    ],
+                    [
+                        3.0,
+                        10.0,
+                        0.7
+                    ],
+                    [
+                        3.0,
+                        10.0,
+                        2.7
+                    ],
+                    [
+                        4.5,
+                        10.0,
+                        2.7
+                    ]
+                ]
+            },
+            "properties": {
+                "type": "ApertureProperties",
+                "energy": {
+                    "type": "ApertureEnergyPropertiesAbridged"
+                }
+            },
+            "aperture_type": {
+                "type": "Window"
```

Comment:
Are the aperture_types:
`Window`
`OperableWindow`
`GlassDoor`
`Skylight`

Reply:
`Skylight` is not an aperture type since you will know if an Aperture is a skylight if its parent face is a `RoofCeiling` or `Floor`.  The other three are all aperture types and, yes, those are the only 3.